extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/extractors/YoutubeChannelVideosExtractor.java

```diff
@@ -0,0 +1,118 @@
+package org.schabi.newpipe.extractor.services.youtube.extractors;
+
+import com.grack.nanojson.JsonArray;
+import com.grack.nanojson.JsonObject;
+
+import org.schabi.newpipe.extractor.InfoItem;
+import org.schabi.newpipe.extractor.MixedInfoItemsCollector;
+import org.schabi.newpipe.extractor.StreamingService;
+import org.schabi.newpipe.extractor.channel.ChannelTabExtractor;
+import org.schabi.newpipe.extractor.downloader.Downloader;
+import org.schabi.newpipe.extractor.exceptions.ExtractionException;
+import org.schabi.newpipe.extractor.exceptions.ParsingException;
+import org.schabi.newpipe.extractor.linkhandler.ListLinkHandler;
+import org.schabi.newpipe.extractor.localization.TimeAgoParser;
+
+import java.io.IOException;
+
+import javax.annotation.Nonnull;
+
+import static org.schabi.newpipe.extractor.services.youtube.linkHandler.YoutubeParsingHelper.getJsonResponse;
+
+public class YoutubeChannelVideosExtractor extends ChannelTabExtractor {
```

Comment:
The id is being extracted using the default way (`ListLinkHandler`) which returns a string starting in `user/` or `channel/`. Was this intentional?

But I guess it doesn't matter that much anyway because, currently, a tab can only be created by having the `ChannelInfo` first, let's see how the sort order can be implemented later.

Also, I see that the name is being used as a [sort of an id](https://github.com/TeamNewPipe/NewPipe/blob/a1a9b54432ae922dad144c3319e37a42fc65ec5b/app/src/main/java/org/schabi/newpipe/fragments/detail/ChannelDetailFragment.java#L544-L567) right now.

Reply:
No, I just couldn't think of any other way, as a `LinkHandler` isn't really needed, since it could only be get through the `ChannelInfo`/`ChannelExtractor`, which is intentional.